python/sqlflow_submitter/tensorflow/predict.py

```diff
@@ -50,35 +50,15 @@ def define_tf_flags():
 class FastPredict:
     def __init__(self, estimator, input_fn):
         self.estimator = estimator
-        self.first_run = True
-        self.closed = False
         self.input_fn = input_fn
 
-    def _create_generator(self):
-        while not self.closed:
```

Comment:
@typhoonzero I don't understand why we need the `while loop`. I guess is that `len(feature_batch) == 2` BUG below led the implementer add it.